### PR TITLE
DAOS-7493 test: Skip offline extend (oclass) test

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-large: pr,hw,large offline_extend

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

**Comment by rpadma2** [comment]
This is PR is ready for landing once we get the approval. The offline extend (oclass) test is skipped.

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**
